sarif-support-for-code-scanning.md:`runAutomationDetails` fix

SwuduSusuwu opened the pull request:
The previous example was "2021-02-01 == February 2, 2021" which is not normal calendar use. The new example fixes this.

<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we can triage your pull request to the best possible team for review.
-->

### Why:

Closes: https://github.com/github/docs/issues/34819

<!-- If there's an existing issue for your change, please link to it above.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed (if available, include any code snippets, screenshots, or gifs):

<!-- Let us know what you are changing. Share anything that could provide the most context.
If you made changes to the `content` directory, a table will populate in a comment below with links to the preview and current production articles. -->

### Check off the following:

- [ X ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ X ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

SwuduSusuwu committed:
sarif-support-for-code-scanning.md:`runAutomationDetails` fix

The previous example was "2021-02-01 == February 2, 2021" which is not normal calendar use.
The new example fixes this.

nguyenalex836 commented:
@SwuduSusuwu Thank you for opening a PR! I believe this doc is following [ISO date format](https://www.w3.org/QA/Tips/iso-date#:~:text=The%20ISO%20date%20format,the%20day%20%5B01%20to%2031%5D), which should actually mean `2021-02-01 == February 1, 2021`!

I'll get this triaged for review, just so another member of the team can confirm 💛

subatoi commented on content/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning.md:
```diff
@@ -239,11 +239,11 @@ Use the category to distinguish between multiple analyses for the same tool or c
 | my-analysis for tool1 | None | my-analysis for tool1
 
-* The run with an `id` of "my-analysis/tool1/2021-02-01" belongs to the category "my-analysis/tool1". Presumably, this is the run from February 2, 2021.
+* The run with an `id` of "my-analysis/tool1/2021-02-01" belongs to the category "my-analysis/tool1". Presumably, this is the run from January 2, 2022.
```
```suggestion
* The run with an `id` of "my-analysis/tool1/2021-02-01" belongs to the category "my-analysis/tool1".
```

subatoi reviewed (dismissed):
Hi @SwuduSusuwu and thanks for raising a PR. You're right about the mistake.

Instead of just changing the written part to match the correct date, we should delete it altogether: different countries use different date systems and not everyone follows ISO8601, sadly. Stating the date doesn't add anything, either.

Once we resolve that point we can merge this on October 7th, after our freeze ends. Thank you!

SwuduSusuwu committed:
"note: ... your system locale can replace this"

After calendar example, has "(note: this ISO8601 format is used in Europe + US, but your system locale can replace this)"

SwuduSusuwu committed:
Update sarif-support-for-code-scanning.md, remove trailing spaces from last commit

Removes trailing spaces from last commit

SwuduSusuwu committed:
Update sarif-support-for-code-scanning.md Reuse suggestion

Partial reuse (removes ambigious language from this)

subatoi commented:
Hi, I appreciate what you're trying to say with the new changes, but we'd be better off keeping this simple and simply delete the line I noted in my last suggestion.

nguyenalex836 commented:
@SwuduSusuwu Hello! 👋 Just wanted to make sure you saw @subatoi's last comment. Once you've deleted the note that he referenced, we'll be happy to get his merged 💛

nguyenalex836 committed:
Remove ISO8601 date format note

nguyenalex836 committed:
Merge branch 'main' into patch-1

nguyenalex836 commented:
@SwuduSusuwu I went ahead and removed the line noted in @subatoi's suggestion 💛 We'll go ahead and get this merged - thank you for raising a flag on this issue, and contributing to GitHub Docs! ✨

nguyenalex836 committed:
Merge branch 'main' into patch-1
